transports/smallwebrtc/README.md

```diff
@@ -0,0 +1,116 @@
+TODO: fix it
+
+<h1><div align="center">
+ <img alt="pipecat client react native" width="500px" height="auto" src="https://raw.githubusercontent.com/pipecat-ai/pipecat-client-react-native-daily-transport/main/pipecat-react-native.png">
+</div></h1>
+
+[![Docs](https://img.shields.io/badge/documentation-blue)](https://docs.pipecat.ai/client/introduction)
+![NPM Version](https://img.shields.io/npm/v/@pipecat-ai/react-native-daily-transport)
+
+The Pipecat client React Native Transport library exports a `RNSmallWebRTCTransport` that has the [SmallWebRTCTransport](https://docs.pipecat.ai/client/js/transports/small-webrtc) transport associated.
+
+## Minimum OS/SDK versions
+
+This package introduces some constraints on what OS/SDK versions your project can support:
+
+- **iOS**: Deployment target >= 15
+- **Android**: `minSdkVersion` >= 24
+
+## Installation
+
+Install `@pipecat-ai/react-native-small-webrtc-transport`.
+
+```bash
+npm i @pipecat-ai/react-native-small-webrtc-transport
+```
+
+If you are using the [DailyMediaManager](../../libs/daily-media-manager/README.md), you will also need to install it along with its peer dependencies:
+
+```bash
+npm i @pipecat-ai/react-native-daily-media-manager
+npm i @daily-co/react-native-daily-js@^0.82.0
+npm i @daily-co/react-native-webrtc@^124.0.6-daily.1
+npm i @react-native-async-storage/async-storage@^1.24.0
+npm i react-native-background-timer@^2.4.1
+npm i react-native-get-random-values@^1.11.0
+```
+
+If you are using Expo, you will also need to add the following dependencies:
+
+```bash
+npm i @daily-co/config-plugin-rn-daily-js@0.0.10
+```
+
+All the details about Expo can be found [here](https://github.com/daily-co/rn-daily-js-expo-config-plugin).
+
+A full demo can be found [here](https://github.com/pipecat-ai/pipecat-examples/tree/main/p2p-webrtc/video-transform/client/react-native)
+
+## Quick Start
+
+Instantiate an `RTVIClient` instance, wire up the bot's audio, and start the conversation:
+
+```typescript
+const options: SmallWebRTCTransportConstructorOptions = {
+  mediaManager: new DailyMediaManager(),
+};
+let pipecatClient = new PipecatClient({
+  transport: new RNSmallWebRTCTransport(options),
```

Comment:
Hm, do we actually want/need this `RN` prefix?

I know that `SmallWebRTCTransport` already exists for web, so I suppose if you were a user with a cross-platform codebase prefixes could be useful, but having such a codebase seems like the rare case rather than the dominant case, and even then the user could just rename on import.